architecture/core_objects/routing.adoc

```diff
@@ -299,30 +307,32 @@ link:#special-notes[special notes] below.
 [[wildcard-certificates]]
 *Wildcard Certificates*
 
-Based on the implementation, you may be able to use a default certificate. Default certificates
-are useful for implementing a wildcard certificate for the router.  For example, if you have
-many routes that end in example.com you may wish to install a router with a wild card
-certificate for `*.example.com`.
+Based on the implementation, you can use a default certificate. Default
+certificates are useful for implementing a wildcard certificate for the router.
+For example, if you have many routes that end in example.com you may wish to
+install a router with a wildcard certificate for `*.example.com`.
 
-To provide the default certificate to the router you must specify it in the create command with
-the default-cert option. The certificate should be a concatenated file of the key, certificate,
-and any CA certificates that are required by the browser. The certificate should be in a
-form acceptable by the underlying router implementation. In the case of HAProxy it should be a
-PEM based certificate.
-
-****
-`oadm router --credentials="$KUBECONFIG" --default-cert=/full/path/to/certificate.pem`
-****
+To provide the default certificate to the router you must specify it in the
+create command with the `--default-cert` option. The certificate should be a
+concatenated file of the key, certificate, and any CA certificates that are
+required by the browser. The certificate must be in a form acceptable by the
+underlying router implementation. In the case of HAProxy it should be a PEM
+based certificate.
 
-For HAProxy, if a default certificate is provided, it will load it first. The certificate that
-is loaded first will be presented to any route that matches the CN on the certificate and
-any route that is secure but does not match any configured certificates. For example, if
-the default certificate is for `\*.example.com` and a secure route for `www.foo.com` is created
-with no certificates the route will still be written and the router will serve the `*.example.com`
-certficiate. This may result in a browser warning for users since the CN on the certificate
-does not match the url.
+----
+$ oadm router --credentials="$KUBECONFIG"
+--default-cert=/full/path/to/certificate.pem
+----
 
```

Comment:
And while backslashes and multiple lines are super appropriate for long commands themselves (readability, copy/paste-ability), the scrollbars are still relevant so that if there is ever large command output or we have to show an example file that has some longer lines, the scrollbars will do their thing. Best of both worlds.